cmd/list.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"text/tabwriter"
+
+	"github.com/cypherpunkarmory/punch/restapi"
+	"github.com/spf13/cobra"
+)
+
+var listCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List your subdomains",
+	Long:  "List subdomains you have previously reserved and also subdomains that are currently in use by you.",
```

Comment:
"List subdomains you have previously reserved or that are currently in use by you."